pyexasol/http_transport.py

```diff
@@ -30,16 +31,16 @@ class SqlQuery:
     # set these values in param dictionary to ExaConnection
     column_delimiter: Optional[str] = None
     column_separator: Optional[str] = None
-    columns: Optional[list[str]] = None
+    columns: Optional[Iterable[str]] = None
     comment: Optional[str] = None
-    csv_cols: Optional[list[str]] = None
+    csv_cols: Optional[Iterable[str]] = None
     encoding: Optional[str] = None
     format: Optional[str] = None
     null: Optional[str] = None
     row_separator: Optional[str] = None
 
     def _build_csv_cols(self) -> str:
-        if not self.csv_cols:
+        if self.csv_cols is None:
```

Comment:
What happens if the iterables where set, but are empty? i think this would return false. is that what you want?